vaadin-combo-box-flow-demo/src/main/java/com/vaadin/flow/component/combobox/demo/ComboBoxView.java

```diff
@@ -254,29 +259,101 @@ private void storingCustomValues() {
     private void lazyLoading() {
         //@formatter:off
         // begin-source-example
-        // source-example-heading: Lazy loading with callbacks
+        // source-example-heading: Lazy Loading with Callback
+        // PersonService can be found:
+        // https://github.com/vaadin/vaadin-combo-box-flow/tree/master/vaadin-combo-box-flow-demo/src/main/java/com/vaadin/flow/component/combobox/demo/service/PersonService.java
+
+        ComboBox<Person> comboBox = new ComboBox<>();
+        PersonService service = new PersonService(500);
```

Comment:
Not done yet. Still in progress...

Reply:
I reverted the changes in `PersonService` so as to retain initial API and exclude Vaadin-specific `Query` class from it. But due to a `setItems` clash, I was forced to use a lambda, instead of method reference (`service::fetch`), which probably reduce the readability and code 'sugar'.